### PR TITLE
Fixed position of the host info (name and logo)

### PR DESCRIPTION
**Pull request by miniche**

### CODE REVIEW

File: centralreport/web/css/style-responsive.css [hunk 1]
```diff
@@ -27,6 +27,13 @@
         margin-top: 0;
     }
 
+    .host_name {
+        text-align: center;
+    }
+    .host_logo {
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css]:
``` css
.host_name,
.host_logo {
    text-align: center;
}
```

File: centralreport/web/css/style-responsive.css [hunk 2]
```diff
@@ -41,6 +48,13 @@
         -moz-box-shadow: none;
         box-shadow: none;
     }
+
+    .host_name {
+        text-align: left;
+    }
+    .host_logo {
+        text-align: left;
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css:52]: Same thing here.

File: centralreport/web/css/style-responsive.css [hunk 3]
```diff
@@ -55,6 +69,12 @@
         box-shadow: 0 1px 2px rgba(0, 0, 0,.15);
     }
 
+    .host_name {
+        text-align: left;
+    }
+    .host_logo {
+        text-align: left;
+    }
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css:71]: Same thing here.

File: centralreport/web/css/style-responsive.css [hunk 4]
```diff
@@ -70,6 +90,12 @@
         box-shadow: 0 1px 2px rgba(0, 0, 0, .15);
     }
 
+    .host_name {
+        text-align: left;
+    }
+    .host_logo {
+        text-align: left;
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css:89]: Same thing here.

File: centralreport/web/css/style-responsive.css [hunk 5]
```diff
@@ -84,5 +110,11 @@
...
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css:107]: Same thing here.

File: centralreport/web/css/style.css [hunk 6]
```diff
@@ -11,6 +11,8 @@
  */
 html, body {
     background-color: #eee;
+    padding-right: 0px;
+    padding-left: 0px;
```
Comment (Ninir) [inline centralreport/web/css/style.css]: When you have a `0` value, you don't need the `px` suffix

File: centralreport/web/css/style-responsive.css (hunk 1 above)
Comment (Guimove) [inline centralreport/web/css/style-responsive.css]: Ok with Ninir

File: centralreport/web/css/style-responsive.css (hunk 2 above)
Comment (Guimove) [inline centralreport/web/css/style-responsive.css:52]: Same

File: centralreport/web/css/style-responsive.css (hunk 3 above)
Comment (Guimove) [inline centralreport/web/css/style-responsive.css:71]: Same same

File: centralreport/web/css/style-responsive.css (hunk 4 above)
Comment (Guimove) [inline centralreport/web/css/style-responsive.css:89]: Always the same

File: centralreport/web/css/style-responsive.css (hunk 5 above)
Comment (Guimove) [inline centralreport/web/css/style-responsive.css:107]: Bored of writing always same things

File: centralreport/web/css/style-responsive.css [hunk 7]
```diff
@@ -41,6 +46,11 @@
         -moz-box-shadow: none;
         box-shadow: none;
     }
+
+    .host_name,.
```
Comment (Ninir) [inline centralreport/web/css/style-responsive.css]: Error (The dot should be before host_logo)

File: centralreport/web/css/style-responsive.css [hunk 8]
```diff
@@ -41,6 +46,11 @@
...
+    host_logo {
+        text-align: left;
+    }
```
Comment (Guimove) [inline centralreport/web/css/style-responsive.css:53]: dot place mistake

**Pull request merged**